maven-artifact/src/main/java/org/apache/maven/artifact/versioning/ComparableVersion.java

```diff
@@ -354,15 +354,15 @@ public String toString()
         implements Item
     {
         private static final List<String> QUALIFIERS =
-                Arrays.asList( "alpha", "beta", "milestone", "rc", "snapshot", "", "sp"  );
+                Arrays.asList( "alpha", "beta", "milestone", "preview", "rc", "snapshot", "", "sp"  );
 
         private static final Properties ALIASES = new Properties();
         static
         {
-            ALIASES.put( "ga", "" );
+            ALIASES.put( "cr", "rc" );
             ALIASES.put( "final", "" );
+            ALIASES.put( "ga", "" );
             ALIASES.put( "release", "" );
-            ALIASES.put( "cr", "rc" );
         }
```

Comment:
What is the benefit of this reorder? None, no?

Reply:
not functional but makes the tag ordering intention clearer